src/magicbox/SuggestionsManager.ts

```diff
@@ -19,11 +19,14 @@ export interface SuggestionsManagerOptions {
   timeout?: number;
 }
 
+type direction = 'up' | 'down' | 'left' | 'right';
```

Comment:
I think we usually capitalise the first letter of our types, but I would check in the project and follow the convention.

Reply:
From what I saw, you are right.
I will capitalize it